include/swift/Serialization/ModuleFormat.h

```diff
@@ -1064,6 +1064,7 @@ namespace decls_block {
     AccessLevelField, // setter access, if applicable
     DeclIDField, // opaque return type decl
     BCFixed<2>,  // # of property wrapper backing properties
+    BCFixed<8>, // total number of vtable entries introduced by all accessors
```

Comment:
`BCVBR<4>` recommended

Reply:
I remember you mentioned that and I had forgotten about it. Thanks for the reminder!